app/quests/migrations/0028_auto_20201109_2213.py

```diff
@@ -4,7 +4,8 @@
 
 
 def forwards(apps, schema_editor):
-    from quests.models import Quest
+    Quest = apps.get_model('quests', 'Quest')
```

Comment:
niiiiice

Reply:
that one was a hard find and the reason why people wasn't able to build locally.